addons/web/static/src/legacy/scss/fields.scss

```diff
@@ -51,6 +51,16 @@
     }
 }
 
+// Invalid background
+// Avoid overlapping the bg in the input inside the list view nor on
+// the order_line because the o_invalid_cell already has the background
```

Comment:
> the order_line because...

Specific customisations (if needed) should be located in the module itself,  not in web ;)